ais/api/serializers.py

```diff
@@ -103,7 +104,8 @@ def model_to_data(self, address):
 
         if address.geocode_type:
             from shapely.geometry import Point
-            shape = Point(address.geocode_x, address.geocode_y)
+            if self.in_street: print("TRUE")
+            shape = Point(address.geocode_street_x, address.geocode_street_y) if self.in_street else Point(address.geocode_x, address.geocode_y)
```

Comment:
how's this:

parcel_source_geocode_types = ("dor_street", "dor_parcel", "pwd_street", "pwd_parcel")
        if address.geocode_type in parcel_source_geocode_types: